Source/Rhetos.Deployment/ApplicationGenerator.cs

```diff
@@ -73,7 +73,7 @@ public class ApplicationGenerator
             _deployOptions = deployOptions;
         }
 
-        public void ExecuteGenerators()
+        public void ExecuteGeneratorsAndUpdateDatabase()

```

Comment:
ExecuteGeneratorsAndUpdateDatabase should probably just call ExecuteGenerators and UpdateDatabase?